client/my-sites/plugins/plugins-dashboard/index.tsx

```diff
@@ -151,7 +152,10 @@ const PluginsDashboard = ( {
 	const sitesWithoutPluginAvailable = sitesToShow.filter(
 		( site ) =>
 			! sitesWithPlugin.find( ( siteWithPlugin ) => siteWithPlugin?.ID === site?.ID ) &&
-			! ( isJetpackCloud() && hasMarketplaceProduct( productsList, pluginSlug ) )
+			! (
+				( isJetpackCloud() || isA8CForAgencies() ) &&
```

Comment:
How about we create a constant, something like:

`const isJetpackCloudOrA8CForAgencies =  isJetpackCloud() || isA8CForAgencies()`